vocabularies/Offline.xml

```diff
@@ -49,7 +49,21 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
            attempt to upload any changes that are made to them locally.
            The service as implemented on the server is described by the CSDL document if
            all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+        </Annotation>
+      </Term>
+
+      <Term Name="LocalDraft" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated model is draft and exists only on client devices. A draft entity
+        can be converted to non-draft entity for uploading"/>
+         <Annotation Term="Core.LongDescription">
+           <String>Typically for each entity type there is one corresponding entity set, however OData permits having multiple entity sets sharing the same entity type.
+            This can be taken advantage of to enable a facility for local (offline) drafts by defining an additional (draft) entity set for an existing entity type.
+            For example, with existing entity type "Order" and existing entity set "Orders", an additional "DraftOrders" entity set could be defined, annotated with
+            this annotation. When a client application creates an entity instance, it could be created in the "DraftOrders" entity set. A subsequent offline entity
+            upload operation should omit the uploading of any local draft entities. The DataService.activateDraft API can be used to transition a draft entity
```

Comment:
Where is the `DataService.activateDraft` API defined/documented? Please add a link

Reply:
API reference removed as suggested, thanks a lot for the advice.